addons/binding/org.openhab.binding.mqtt/src/main/java/org/openhab/binding/mqtt/handler/BrokerHandler.java

```diff
@@ -179,40 +178,40 @@ protected void assignSSLContextProvider(BrokerHandlerConfig config, MqttBrokerCo
      */
     protected MqttBrokerConnection createBrokerConnection() throws IllegalArgumentException {
         String host = config.host;
-        if (StringUtils.isBlank(host) || host == null) {
+        if ((host == null) || StringUtils.isBlank(host)) {
             throw new IllegalArgumentException("Host is empty!");
         }
 
-        final MqttBrokerConnection connection = new MqttBrokerConnection(host, config.port, config.secure,
-                config.clientID);
+        MqttBrokerConnection connection = new MqttBrokerConnection(host, config.port, config.secure, config.clientID);
```

Comment:
Why did you remove the `final`? It helps other developers to not accidentally overwrite the object

Reply:
see above